circle.yml

```diff
@@ -1,10 +1,17 @@
-machine:
-  python:
-    version: 3.5.2
-  environment:
-    ANSIBLE_REMOTE_USER: ubuntu
-    PYTHON27: /opt/circleci/python/2.7.12/bin/python
-    TEST_HOSTS: integration/hosts
-test:
-  override:
-    - cd tests && make test
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/python:3.5.4
+        environment:
+          ANSIBLE_REMOTE_USER: ubuntu
+          TEST_HOSTS: integration/hosts
+    steps:
+      - checkout
+      - run:
+          command: sudo apt-get update -qq && sudo apt-get install -qq python-dev
```

Comment:
Out of curiosity, what is python-dev needed for?

Reply:
it was PyYaml